microprofile/messaging/core/src/main/java/io/helidon/microprofile/messaging/UniversalChannel.java

```diff
@@ -69,8 +70,12 @@ void setOutgoing(OutgoingMethod outgoingMethod) {
         this.outgoingMethod = outgoingMethod;
     }
 
-    AtomicBoolean isUp() {
-        return up;
+    AtomicBoolean isLive() {
```

Comment:
This is leaking internal state - this method may just return a `boolean`

Reply:
package private, but true, I've got changes in here from another PR, will fix it there